### PR TITLE
win_sigverif: Fix 'sigverif logs are not created' error 

### PR DESCRIPTION
**Pull request by menli820**
Add command to ensure guest on desktop and wait 60s to make
all popups pop up to eliminate the impact of popups on autoit.

ID:1797847

Signed-off-by: Menghuan Li menli@redhat.com

### CODE REVIEW

File: qemu/tests/win_sigverif.py [hunk 1]
```diff
@@ -22,6 +23,8 @@ def run(test, params, env):
 
     vm = env.get_vm(params["main_vm"])
     session = vm.wait_for_login()
+    vm.send_key('meta_l-d')
+    time.sleep(60)
```
Comment (lijinlijin) [inline qemu/tests/win_sigverif.py]: Shouldn't we wait 60s first, then send key?

**Comment by xiagao** [comment]
> Add command to ensure autoit tool run normally
> 
> BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1797847
> 
> Signed-off-by: Menghuan Li [menli@redhat.com](mailto:menli@redhat.com)

ID:1797847
Pls delete the bz link

**Comment by xiagao** [comment]
And also delete bz link in commit.

**Comment by kimi1978wy** [comment]
delete bug id in commit, and change bz link to id:1797847 in description (comment#0)

**Comment by menli820** [comment]
Test the case for 1000 times without issues.

**Comment by lijinlijin** [comment]
@menli820 Could you describe more details in commit message about your fix;
btw, add case name in the beginning, such as win_sigverif: xxx

**Comment by kimi1978wy** [comment]
Commit description is too long

**Comment by kimi1978wy** [comment]
Test on win8-32,win8.1-64,win2016, all passed
LGTM
Acked by: Yu Wang <wyu@redhat.com>

**Comment by lijinlijin** [comment]
win_sigverif:Fix 'sigverif logs are not created ' error

Please add white space after ":" and delelte space after "created"

**Comment by lijinlijin** [comment]
LGTM
Acked-by: Li Jin lijin@redhat.com

File: qemu/tests/win_sigverif.py (hunk 1 above)
Comment (menli820) [reply qemu/tests/win_sigverif.py]: Actually,the order doesn't matter.

**Comment by YongxueHong** [comment]
Each line of the commit is limited to 72 characters.
Please modify it.
Thanks.

**Comment by menli820** [comment]
@YongxueHong  I have modified it,please help to check it ,much thanks.

**Pull request merged**